src/lib/journey/callbacks/terms-and-conditions/terms-conditions.svelte

```diff
@@ -34,6 +34,5 @@
 </script>
 
 <Checkbox key={inputName} onChange={setValue} value={false}>
-  <!-- TODO: Remove hardcoded text below -->
-  Please accept our below Terms and Conditions
+  <T key="termsAndConditions" />
```

Comment:
I don't see the terms and conditions rendered for this component. Guessing this has not been implemented yet? Also on this point what is our strategy for terms and conditions localization given IDM has its own mechanism for this? (similar to KBA answers)?

Reply:
Oh, and yes, this is very much like the KBA in the fact that some things will come "pre-localized" (if you will) from AM/IDM. This is one of them.